boost-toolfile.spec

```diff
@@ -28,6 +28,15 @@ cat << \EOF_TOOLFILE >%i/etc/scram.d/boost.xml
 </tool>
 EOF_TOOLFILE
 
+# boost_chrono toolfile
+cat << \EOF_TOOLFILE >%i/etc/scram.d/boost_chrono.xml
+<tool name="boost_chrono" version="@TOOL_VERSION@">
+  <info url="http://www.boost.org"/>
+  <lib name="@BOOST_CHRONO_LIB@"/>
+  <use name="boost"/>
```

Comment:
@fwyzard , looks like boost_chrono  needs boost_system
``` ldd libboost_chrono.so | grep boost
libboost_system.so.1.57.0 => not found
```
so please add dependency on ```<use name="boost_system"/>```

can you also please update the version number to 1.2 of this spec?